Core/src/test/java/com/rakuten/tech/mobile/perf/core/EnvironmentInfoSpec.java

```diff
@@ -13,67 +13,34 @@
 import org.junit.Test;
 import org.mockito.Mock;
 import org.mockito.MockitoAnnotations;
-import org.mockito.invocation.InvocationOnMock;
-import org.mockito.stubbing.Answer;
 
 public class EnvironmentInfoSpec {
 
-  @Mock TelephonyManager tm;
-  @Mock ActivityManager am;
-  @Mock Context ctx;
+  @Mock private TelephonyManager telephonyManager;
+  @Mock private ActivityManager activityManager;
+  @Mock private Context context;
```

Comment:
doesn't mockito complain about private members?

Reply:
Oh, really? I didn't see any warnings, but Android Studio warns that it can be private.

I guess all the other classes aren't using private members, so I'll change this back to match.

Reply:
I used to get runtime errors of mockito not being able to inject private members - maybe mockito has changed though.